packages/design-system/src/components/N8nFormInput/FormInput.vue

```diff
@@ -240,7 +240,7 @@ const validationError = computed<string | null>(() => {
 
 	if (error) {
 		if ('messageKey' in error) {
-			return t(error.messageKey, error.options as object);
+			return t(error.messageKey, error.options as string[]);
```

Comment:
could this be better defined in the original type?

Reply:
Updated.